### ISSUE TITLE
Phishing Reports

### ISSUE THREAD

**Issue by u1t-dev** [u1t-dev/OnBoard#6]
um like we really need to fix this at some point
<img width="967" alt="Screenshot 2024-01-31 at 9 17 33 AM" src="https://github.com/acapzine/OnBoard/assets/119457323/9a2d9ef3-f8b6-4d6b-8740-df43333bfd67">

**Comment by acapzine** [comment]
ive already sent 2 reports of it being false
idk if its faker thats making this happen or not, but it thinks we're phishing/stealing passwords

**Comment by acapzine** [comment]
possibly we might have to switch it over *early* to the subdomain [onboard.u1terior.net](https://onboard.u1terior.net) but idk if that will just mark that is phishing as well

**Comment by u1t-dev** [comment]
i don't think that it's getting flagged anymore, i can check on google in a moment

**Comment by HydrogenMacro** [comment]
what is the domain even

**Comment by u1t-dev** [comment]
https://u1t-dev.github.io/OnBoard/
its fixed!
